script.js

```diff
@@ -1,3 +1,132 @@
+const cards = [
+  {
+    image: 'images/Snapshoot-Portfolio.svg',
+    name: 'Tonic',
+    preview: '',
+    code: '',
+    tags: ['html', 'css', 'javascript'],
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+  {
+    image: 'images/Snapshoot-Portfolio-2.png',
+    name: 'Multi-Post Stories',
+    preview: '',
+    code: '',
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+  {
+    image: 'images/Snapshoot-Portfolio-3.png',
+    name: 'Tonic',
+    preview: '',
+    code: '',
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+  {
+    image: 'images/Snapshoot-Portfolio-4.png',
+    name: 'Multi-Post Stories',
+    preview: '',
+    code: '',
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
+];
+
+// projects dynamic
+// const cardsContainer = document.getElementById('cards')
+const cardsContainer = document.querySelector('.portfolio-container');
+cards.forEach((cardContent) => {
+  const item = document.createElement('work-container-main');
+
+  item.classList.add('cards-held-here');
+  // item.classList.add('work-container-desktop');
```

Comment:
- [ ]  Kindly remove all the non descriptive commented code in this file for a cleaner codebase